cwl/src/test/resources/three_step.cwl

```diff
@@ -1,6 +1,15 @@
 cwlVersion: v1.0
 class: Workflow
 id: three_step
+hints:
+  # hints can be anything (CwlAny)
+  - fee: fi
+    fo: fum
+  - eenie: meenie
+    minie: mo
+  - 42
```

Comment:
ToL: I don't know whether the `three_step.cwl` is the best place to be testing the limits of `CwlAny`'s parsing

Reply:
It's not, this is a quick hack. 🙂